Java/core/src/main/java/com/amazon/randomcutforest/state/store/PointStoreState.java

```diff
@@ -46,6 +60,34 @@
      * An array of reference counts for each stored point.
      */
     private short[] refCount;
+    /**
+     * is direct mapping enabled
+     */
+    @Getter
```

Comment:
I think this is redundant since the class is annotated with `@Data`

Reply:
fixed.